.github/workflows/ci.yml

```diff
@@ -67,12 +68,12 @@ jobs:
         uses: actions/checkout@v3
       - name: Run ShellCheck
         run: shellcheck bin/*
-  format:
-    runs-on: macos-latest
+  shfmt:
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Install shfmt
-        run: brew install shfmt
-      - name: Run shfmt
-        run: shfmt -d -i 2 -ci .
+      - uses: reviewdog/action-shfmt@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We probably need to provide this with some extra permissions for it to effectively comment back on the PR

Reply:
According to [[1]](https://github.com/reviewdog/action-eslint/issues/148#issuecomment-1282861097) it should be fine like added [here](https://github.com/jonathanmorley/asdf-pnpm/pull/21/commits/b018ebe33f689ea48ccf83b97f3374e789355a14)

Reply:
Upon further inspection, this warning may not be avoidable https://github.com/reviewdog/reviewdog/blob/2765abc90a9df64b67a2171df9bbea5b3a697267/cmd/reviewdog/main.go#L277

Reply:
Once it's merged, it should be fine, right?

Reply:
Do you have concerns using reviewdog this way?